janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/function/CQLSimpleMutateManyUnloggedFunction.java

```diff
@@ -0,0 +1,99 @@
+// Copyright 2021 JanusGraph Authors
```

Comment:
I've ran a diff between this file and CQLExecutionServiceMutateManyUnloggedFunction and most of the code is identical. Is there a way we could isolate the Future/CompletableFuture diffs so the rest of the code remains shared between classes?